eng/readme-templates/README.md

```diff
@@ -88,6 +88,6 @@ See the [.NET Framework Lifecycle FAQ](https://support.microsoft.com/help/17455/
 
 # License
 
-* Legal Notice: [Container License Information](https://aka.ms/mcr/osslegalnotice)
-
-The .NET Framework images use the same license as the [Windows Server Core base image](https://hub.docker.com/_/microsoft-windows-servercore/).
+* The [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/) applies to all [.NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/).
```

Comment:
I'm going to channel @dleeapho with this comment.  It is not ideal that the first two bullets are complete sentences while the third is not.  What about something like the following?  This format feels more consistent with the other list sections within the readme.

```
* Legal Notice: [Container License Information](https://aka.ms/mcr/osslegalnotice)
* [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/): applies to all [.NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/)
* [Visual Studio Tools license](https://visualstudio.microsoft.com/license-terms/mlt031519/): applies to all [.NET Framework SDK container images](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/)
```

Reply:
Ok, made some updates.  Let me know how that looks.

Reply:
I like that. How about this small diff.

> * [Microsoft Container Images Legal Notice](https://aka.ms/mcr/osslegalnotice): applies to all [.NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/) 
> * [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/): applies to all [.NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/)
> * [Visual Studio Tools license](https://visualstudio.microsoft.com/license-terms/mlt031519/): applies to all [.NET Framework SDK container images](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/)